lib/resources/r4/encounter_patch.yaml

```diff
@@ -64,3 +64,36 @@ operations:
         <li>`Encounter.type` value should be a list.</li>
         <li>When a list is provided, the first value would be used to patch `Encounter.type`.</li>
       </ul>
+
+  - name: replace-extension
+    path: /extension
+    operation: replace
+    type: List of Extension
+    url: http://hl7.org/fhir/encounter-definitions.html#Encounter.extension
+    description: The aray of extension of encounter.
+    example: |
+      {
+        "op": "replace",
+        "path": "/extension",
+        "value":  [
+          {
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/payment-collection-status",
+            "valueCodeableConcept": {
+            "coding": [
```

Comment:
can we fix the indentation here

Reply:
https://github.com/cerner/fhir.cerner.com/commit/d08e9515916e1573d931f14dccab90511f189c90#diff-97d84edc87e609cdbf80d6ad20cbc681R82